### PR TITLE
Updates for tox and travis tests

### PR DESCRIPTION
**Pull request by baylee**
@rmutter for review

Now running tests against multiple versions of python, django, and drf

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    - TOX_ENV=py34-django19-drf33
+    - TOX_ENV=py35-django19-drf31
+    - TOX_ENV=py35-django19-drf32
+    - TOX_ENV=py35-django19-drf33
+
+matrix:
+  fast_finish: true
+
+install:
+  # Virtualenv < 14 is required to keep the Python 3.2 builds running.
```
Comment (rmutter) [inline .travis.yml]: since we're not supporting python 3.2 do we need this?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+language: python
+
+python:
+  - "3.5"
```
Comment (rmutter) [inline .travis.yml]: Is this actually testing each version of python? It looks like the different django and drf versions are being installed properly but it looks like everything is against 3.5 in the travis logs? https://travis-ci.org/yeti/YAK-server/jobs/118313043

**Comment by rmutter** [comment]
@baylee couple of comments! :palm_tree:

File: .travis.yml (hunk 2 above)
Comment (baylee) [inline .travis.yml]: It's working, that's just the way Travis displays it. See https://travis-ci.org/tomchristie/django-rest-framework?branch=master for an example

File: .travis.yml (hunk 1 above)
Comment (baylee) [inline .travis.yml]: The idea is actually to support 3.2, but that's pending on #22

**Pull request merged**